rDNA/inst/WORDLIST

```diff
@@ -80,5 +80,6 @@ Unimodality
 vegdist
 visone
 walktrap
```

Comment:
I think it should always be capitalized.

Reply:
The spell check also looks at options for arguments in the function descriptions afaik. So we have a couple of instances where Walktrap is spelled differently. For example: `\item{"walktrap"}{Walktrap community detection.}`.  So I think it makes sense to leave both versions.

Reply:
Makes sense.